migration/sql-files/006-add-sa-token-encrypted-to-cluster.sql

```diff
@@ -0,0 +1,2 @@
+-- Store sa_token_encrypted for cluster
+ALTER TABLE cluster ADD COLUMN sa_token_encrypted boolean NOT NULL DEFAULT FALSE;
```

Comment:
please don't use `NOT NULL` in this sql statement: if we need to rollback the deployment, we'll be in troubles.

Reply:
I don't see reason to rollback the deployment. Do you see?

Reply:
it's just a good practice IMO